tests/ruby/admin_spec.rb

```diff
@@ -208,6 +208,28 @@
         threads.map(&:join)
         connections.map(&:close)
       end
+
+      it "show correct max_wait" do
+        threads = []
+        connections = Array.new(4) { PG::connect("#{pgcat_conn_str}?application_name=one_query") }
+        connections.each do |c|
+          threads << Thread.new { c.async_exec("SELECT pg_sleep(1.5)") }
+        end
+
+        sleep(2.5) # Allow time for stats to update
```

Comment:
We should introduce a command to refresh stats immediately. I can see our CI getting slower and slower over time as we add features :)